validation/superchain-genesis_test.go

```diff
@@ -16,6 +16,11 @@ import (
 
 func testGenesisHash(t *testing.T, chainID uint64) {
 	skipIfExcluded(t, chainID)
+
+	if chainID == 10 {
+		t.Skip("OP Mainnet has a pre-bedrock genesis, so we exclude it from this check.")
+	}
```

Comment:
I think I finally understand why OP Mainnet doesn't work. It might not be because it has a pre-bedrock genesis, but that the compressed genesis file does contains an empty `alloc: {}` property. 

Likely added using the "alternative" add-chain path 
https://github.com/ethereum-optimism/superchain-registry/blob/9702f3ce28c79d3a9f0ec6f262eb6c2f997298f0/add-chain/cmd/compress-genesis.go#L51-L94